sim/dex-species.ts

```diff
@@ -152,6 +152,10 @@ export class Species extends BasicEffect implements Readonly<BasicEffect & Speci
 	readonly heightm: number;
 	/** Color. */
 	readonly color: string;
+	/** Tags, miscellaneous data.
+	* This just checks if a Pokemon has some sort of Legendary or Mythical status right now though.
+	*/
+	readonly tags?: string[];
```

Comment:
I would fill this out (`this.tags = data.tags || []`) so you don't need the `?.`

Reply:
I would also make it `SpeciesTag[]`

And then have `type SpeciesTag = "Mythical" | "Restricted Legendary";`

Reply:
I've done this. But also added a Sub-Legendary alternative in keeping w/ data

(reason stated below)